.github/workflows/mypy.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Type checking with mypy
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  mypy-run:
```

Comment:
this job can be moved to the `main.yml` workflow and by default (if not `depends_on` will be added to it) it won't have to run on pytest-related jobs

Reply:
Thanks for the tip! Fixed in https://github.com/pytest-dev/pytest-factoryboy/pull/152